instance_selector/widgets.py

```diff
@@ -28,6 +19,12 @@ def __init__(self, model, **kwargs):
         self.choose_another_text = _("Choose another %s") % model_name
         self.link_to_chosen_text = _("Edit this %s") % model_name
 
```

Comment:
If we were to Inherit from `WidgetWithScript` and `widgets.Input`, we'd need to include the following on `init`

```
self.clear_choice_text = _("Clear choice")
self.show_edit_link = True
self.show_clear_link = True
```